cores/arduino/dccm/dccm_alloc.c

```diff
@@ -0,0 +1,42 @@
+/*
+Copyright (c) 2016 Intel Corporation.  All right reserved.
+
+This library is free software; you can redistribute it and/or
+modify it under the terms of the GNU Lesser General Public
+License as published by the Free Software Foundation; either
+version 2.1 of the License, or (at your option) any later version.
+
+This library is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public
+License along with this library; if not, write to the Free Software
+Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+
+*/
+
+#include "dccm_alloc.h"
+
+uint16_t dccm_index = 0;
+
+#ifdef __cplusplus
+ extern "C" {
+#endif
+
+uint8_t* dccm_malloc(uint16_t size)
```

Comment:
Can you return a `void *`, so that it an be used in the same way as malloc?

i.e. `int *ptr = dccm_malloc(5 * sizeof(int));`